### ISSUE TITLE
Create Date-Time Preferences functionality

### ISSUE THREAD

**Issue by nabramovs** [TheOntologist/AGC#12]
Create new tab "Settings". Following options should be available for configuration how events are displayed (and read by Screen Reader) on Events list tab:
1. 12 hour or 24 hour
2. Delimiter between date elements - "-", " " or "/" - default is " ".
3. Show end time for individual events.
4. Do not read month if it is current month
5. Read only start date for full date event (no start time, end date, end time)
6. Show year
7. Months as long name, short name or month number
   8 mm/dd/yy or dd/mm/77 (last is default)

Settings should be saved locally and should not be overwritten by updates. Separate config file or serialization could resolve this.
